src/garage/torch/policies/gaussian_mlp_policy.py

```diff
@@ -1,10 +1,11 @@
 """GaussianMLPPolicy."""
-from torch import nn
 
-from garage.torch.policies import Policy
+import torch
 
+from garage.torch.policies.base import Policy
 
-class GaussianMLPPolicy(nn.Module, Policy):
```

Comment:
I just think we use get_action (or get_actions) method more instead of forward.
So I assume this GaussianMLPPolicy is not a module.

If we need to put `nn.Module` back, I will fix this

Reply:
can we have garage.torch.policies.base.Policy inherit from not only abstract base class, but also nn.Module?
Then we can have gaussianMLPPolicy, and all other new policies inherit from only garage.torch.policies.base.Policy.

Reply:
Hmm I think we have followed the tf tree style to implement DDPG, which I think looks pretty good. Could get back to that? I am sorry for the back-and-forth design decision.

Reply:
Please see #815 .